jwst/lib/tests/test_suffix.py

```diff
@@ -1,12 +1,17 @@
 """Test suffix replacement"""
 
+import logging
 import pytest
 
 from .. import suffix as s
 
+s_logger = logging.getLogger(s.__name__)
+s_logger.addHandler(logging.StreamHandler())
```

Comment:
What is the purpose of adding a logger and handler to the test module?  Is this to capture log messages?  Tests shouldn't be logging anything.

Reply:
The test itself is not logging anything. However, having the module-being-tested logging enabled does provide more info if a test failure occurs.  In this case, it has pointed out some other unrelated issues to be filed.

But, glad this was pointed out, because what the final form of this test was to also turn off the logging once tests are done. Next on the list.